src/components/GodBar/_ThemeSwitcherAlt.vue

```diff
@@ -0,0 +1,162 @@
+<script setup lang="ts">
```

Comment:
whats alt vs non alt

Reply:
<select> element (dropdown) vs the new button type. Might loop back and rename both components.